module.js

```diff
@@ -90,11 +92,11 @@ M.mod_equella.display_equella = function(Y, equellaContainer, width, minheight,
 	generate_html(true);
 	var panel = new Y.Panel({
 	    srcNode : '#' + equellaContainer,
-	    width : width,
+	    width : initialwidth,
```

Comment:
Ah, hold on a second. If we simply stop using `width` here then we end up with an unused variable which is passed into the constructor.

Maybe what also needs to be done here is for `width` to be refactored to `minWidth` (just like there's already `minHeight`) and then do the same check against the calculated `initialwidth` to ensure it's at least `minWidth`.